README.md

```diff
@@ -145,6 +145,11 @@ Simple bootstrap project created by *quarkus-maven-plugin*
 This module covers some basic scenarios around reactive routes in general and also:
 - Validation on request params, request body and responses.
 
+### `servlets`
```

Comment:
Move this section where the rest of http modules are placed. Also, the section name should be `servlet-undertow`.